pilot/pkg/model/test/mockopenidserver.go

```diff
@@ -37,10 +37,16 @@ var (
 
 const (
 	// JwtPubKey1 is the response to 1st call for JWT public key returned by mock server.
-	JwtPubKey1 = "fakeKey1"
+	JwtPubKey1 = "{ \"keys\": [ { \"kid\": \"fakeKey1_1\", \"alg\": \"RS256\", \"kty\": \"RSA\", \"n\": \"abc\", \"e\": \"def\" }," +
```

Comment:
Can you use the literal string to make this more readable, then you don't need to escape every `"` in the string:
```go
jwtPubKey1 = `
{
  "keys": [ { "kid": ... } ],
}
`
```

Reply:
Yup, will do.